tickit/adapters/interpreters/command/command_interpreter.py

```diff
@@ -84,16 +93,33 @@ async def handle(
                 indicating whether an interrupt should be raised by the adapter.
         """
         for _, method in getmembers(adapter):
-            command = getattr(method, "__command__", None)
+            message = message
+            command = cast(Command, getattr(method, "__command__", None))
             if command is None:
                 continue
-            args = command.parse(message)
-            if args is None:
+            parse_result = command.parse(message)
+            if parse_result is None:
+                continue
+            (
+                match_groups,
+                _,
+                match_end,
+                message_end,
+            ) = parse_result
+            if match_end != message_end:
+                # We want the whole (formatted) message to match a command
```

Comment:
Documentation of the whole / complete would be good to have in the class docstring